tailcfg/tailcfg.go

```diff
@@ -1940,6 +1943,10 @@ type SSHAction struct {
 	// AllowLocalPortForwarding, if true, allows accepted connections
 	// to use local port forwarding if requested.
 	AllowLocalPortForwarding bool `json:"allowLocalPortForwarding,omitempty"`
+
+	// SessionHauling, if non-empty, is the IP address of a peer to stream this
+	// SSH session's logs to.
+	SessionHauling string `json:"sessionHauling,omitempty"`
```

Comment:
this name sounds like a bool. How about `SessionHaulDest`?

Also, I think we should use a node stable ID here instead of IP address, lest the source not have that address family available. See `func peerAPIBase`.

Reply:
Or `SessionHaulTargetNode`.

Reply:
@twitchyliquid64 pointed out that we might want to run the aggregator standalone in the future (maybe behind a subnet router), so being able to select a specific IP:Port that isn't a Tailscale node could be useful for that.

But we could also just change this setting (or add a new one) in the future to handle that, so I'm fine with changing both the name and the type here.

Reply:
We'd need a new field to handle a `:port` later anyway, so let's add a new field later if/when we do something else.

Reply:
Updated to use a `tailcfg.StableNodeID` and updated name.